### PR TITLE
Migrate Android CI from Travis CI to GitHub Actions workflows 

### PR DESCRIPTION
**Pull request by jvff**
This PR is based on some initial experimentation with GitHub Actions. I've managed to create three workflows that are roughly equivalent to the previous Travis CI job. I configured them so that they only run when Android relevant files change, so they should only be triggered when it's appropriate.

I'm still learning things, so we can probably tweak this a lot, but hopefully it's good enough to start migrating from Travis.

For building WireGuard-Go for Android, I tweaked the build script a bit so that the architectures to build for can be configured through an environment variable so that we don't waste time building for more than one architecture on CI.

Git checklist:

* [x] Describe the change in **`CHANGELOG.md`** under the `[Unreleased]` header.
* [x] Check that commits follow the [Mullvad coding guidelines](https://github.com/mullvad/coding-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)

### CODE REVIEW

**Review by pinkisemils — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)* status: 0 of 5 files reviewed, all discussions resolved

___
*[wireguard/libwg/build-android.sh, line 15 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469#-MTNzDYR8r4MG01ppfud:-MTNzDYR8r4MG01ppfue:b-4vcf8) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/4545a2045c2377ead9068589288ad109e3f369a8/wireguard/libwg/build-android.sh#L15)):*
> ```Shell
> if [ -z "$ARCHITECTURES" ]; then
>     ARCHITECTURES="arm arm64 x86_64 x86"
> fi
> ```

This could be `ARCHITECTURES=${ARCHITECTURES:-"arm arm64 x86_64 x86"}`

File: .github/workflows/android-xml-tidy.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+    xml-tidy:
+        runs-on: ubuntu-latest
+        steps:
+            # Checkout repository
+            - uses: actions/checkout@v2
+
+            # Check formatting
+            - run: |
+                sudo apt-get install tidy
```
Comment (axti) [inline .github/workflows/android-xml-tidy.yml:19]: Will it download each time from remote repo or from the mirror? Will it cost for downloading from remote repo?

**Review by axti — APPROVED** [review]
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)* status: all files reviewed, 1 unresolved discussion (waiting on @jvff)

**Comment by jvff** [comment]
___
*[.github/workflows/android-xml-tidy.yml, line 19 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469#-MTNMvZX2j4RGjEgiato-r1-19:-MT_faUOAalUR50LEdcR:bleoi66) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/4545a2045c2377ead9068589288ad109e3f369a8/.github/workflows/android-xml-tidy.yml#L19)):*
<details><summary><i>Previously, axti (Alex Granin) wrote…</i></summary><blockquote>

Will it download each time from remote repo or from the mirror? Will it cost for downloading from remote repo?
</blockquote></details>

It will download every execution, but as far as I understood it's only charged by minutes of execution. This should only take a few seconds, so it shouldn't be a big deal. Also, I think that since our repository is public, we are allowed to use GitHub Actions for the repository for free.

At least that's what I understood from here: https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions

What I still don't know is if a job takes less than a minute (say, 20 seconds), then is it rounded up to one minute? Shouldn't matter for now, as long as it's free for our repo, but I'm still curious :/

**Review by axti — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)* status: :shipit: complete! all files reviewed, all discussions resolved

**Review by jvff — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)* status: 4 of 5 files reviewed, all discussions resolved (waiting on @axti)

___
*[wireguard/libwg/build-android.sh, line 15 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469#-MTNzDYR8r4MG01ppfud:-MTa2EgT5uiltri15rr5:boyexb3) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/4545a2045c2377ead9068589288ad109e3f369a8/wireguard/libwg/build-android.sh#L15)):*
<details><summary><i>Previously, pinkisemils (Emīls Piņķis) wrote…</i></summary><blockquote>

This could be `ARCHITECTURES=${ARCHITECTURES:-"arm arm64 x86_64 x86"}`
</blockquote></details>

Done. Thanks!

**Review by faern — APPROVED** [review]
> Reviewed 4 of 5 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2469)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**